tst/Pester.RSpec.InNewProcess.ts.ps1

```diff
@@ -58,4 +58,102 @@ i -PassThru:$PassThru {
             }
         }
     }
+
+    b "Exit codes" {
+
+        t "Exitcode is set to 0 without exiting the process when tests pass, even when some executable fails within test" {
+            $temp = [IO.Path]::GetTempPath()
+            $testpath = Join-Path $temp "$([Guid]::NewGuid().Guid).tests.ps1"
+            $powershell = (Get-Process -Id $pid).Path
+
+            try {
+                $c = "
+                Describe 'd' {
+                    It 'i' {
+                        # an executable exits with 9999 (we use powershell as the executable, because we know it will work cross platform)
+                        & '$powershell' -Command { exit 9999 }
+                        `$LASTEXITCODE | Should -Be 9999
```

Comment:
```suggestion
                        # an executable exits with 99 (we use powershell as the executable, because we know it will work cross platform)
                        & '$powershell' -Command { exit 99 }
                        `$LASTEXITCODE | Should -Be 99
```
Unix exitcodes goes from 0-255. Resets to 0 at every 256.
```
& '/usr/share/powershell/pwsh' -Command { exit 256 }; $LASTEXITCODE
0

#This is the exit code return in the current sample
9999%256
15
```